### ISSUE TITLE
Test case: Users can successfully customize the saving location of downloaded content.

### ISSUE THREAD

**Issue by tracywalker** [mozilla/iris#3145]
https://testrail.stage.mozaws.net/index.php?/cases/view/143567

**Comment by pavelciapa** [comment]
We can set specific folder to download, but Steps 5 and 8 ask to check that download was to certain folder. Shall we check this?

**Comment by tracywalker** [comment]
Yes check that the file was saved to the location (folder) selected for the download location.
